dev/readthedocs.md

```diff
@@ -4,28 +4,43 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+   python manage.py ingest jhu --years 2021 --terms fall
 
 2. How do I then load those courses into my database?
+   python manage.py digest jhu --years 2021 --terms fall
```

Comment:
You don't have to specify the terms/years, you can just do `python manage.py digest jhu`

Reply:
Get it!